modules/sdk/project-templates/project-templates-activator/bnd.bnd

```diff
@@ -2,6 +2,7 @@ Bundle-Description: Creates a Liferay module project that customizes the startin
 Bundle-Name: Liferay Project Templates Activator
 Bundle-SymbolicName: com.liferay.project.templates.activator
 Bundle-Version: 1.0.8
+Liferay-Versions: [7,8)
```

Comment:
Be careful, SF won't like this new header! You have to update SF to allow it, and also to avoid splitting the range in 3 lines (like it did when I ran it locally).